c_src/libsrtp/srtp.c

```diff
@@ -127,42 +127,62 @@ UNIFEX_TERM create(UnifexEnv *env) {
   return create_result(env, state);
 }
 
-UNIFEX_TERM add_stream(UnifexEnv *env, UnifexState *state, int ssrc_type,
-                       unsigned ssrc, UnifexPayload **keys,
-                       unsigned int keys_length, UnifexPayload **keys_mkis,
-                       unsigned int keys_mkis_length, char *rtp_crypto_profile,
-                       char *rtcp_crypto_profile, unsigned window_size,
-                       int allow_repeat_tx) {
+bool init_policy(UnifexEnv *env, srtp_policy_t *policy, int ssrc_type,
+                 unsigned ssrc, UnifexPayload **keys, unsigned int keys_length,
+                 UnifexPayload **keys_mkis, unsigned int keys_mkis_length,
+                 char *rtp_crypto_profile, char *rtcp_crypto_profile,
+                 unsigned window_size, int allow_repeat_tx, UNIFEX_TERM *uerr) {
```

Comment:
`uerr` -> `unifex_error`. BTW, you can return that error and return NULL if there's no error

Reply:
I was thinking about it but I decied to follow the convention in this file. We use in some places variables like `serr`, `err` and `uerr` and also `create_master_keys_array` returns bool instead of pointer so I don't know if we want to fix this in all places or leave it as it is or fix only my changes?

Reply:
Not returning bool is only a suggestion, It can stay as is. Regarding `uerr`, I think we shouldn't use abbreviations when their meaning is not obvious (like in the case of `err`). It can be `unifex_err` either.